### ISSUE TITLE
Cannot add multiple component ids

### ISSUE THREAD

**Issue by MisinformedDNA** [MicrosoftDocs/visualstudio-docs#762]
When I run `vs_installer.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community" --add Microsoft.Net.Component.4.6.TargetingPack` or `vs_installer.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community" --add Microsoft.VisualStudio.Component.PortableLibrary`, I get to the next screen successfully. However, if I want to combine them with ";" as in the docs above, then I get an error. `vs_installer.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community" --add Microsoft.Net.Component.4.6.TargetingPack;Microsoft.VisualStudio.Component.PortableLibrary` results in

> The following IDs could not be installed...
> - **Microsoft.Net.Component.4.6.TargetingPack;Microsoft.VisualStudio.Component.PortableLibrary**: No matching workload or component found.

Why isn't this working?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e97b834-fa82-6aa0-5913-4d5a8ed23366
* Version Independent ID: e988e3ea-8565-ca88-6ade-ab67425a51ae
* Content: [Use command-line parameters to install Visual Studio - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio)
* Content Source: [docs/install/use-command-line-parameters-to-install-visual-studio.md](https://github.com/Microsoft/visualstudio-docs/blob/master/docs/install/use-command-line-parameters-to-install-visual-studio.md)
* Service: **unspecified**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by cpdohert** [comment]
You're using the wrong syntax.  You need --add for *each* componentid you want to install.  The ; syntax is for the includeRecommended and includeOptional arguments (which you aren't using).  The command line you want is

vs_installer.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community" --add Microsoft.Net.Component.4.6.TargetingPack --add Microsoft.VisualStudio.Component.PortableLibrary

**Comment by MisinformedDNA** [comment]
The documentation needs to be improved then. There are no examples of adding multiple component ids and the documentation `--add <one or more workload or component IDs>` sure seems to imply you can just list them after the `add`.

**Comment by MisinformedDNA** [comment]
Yea, I missed that, repeatedly.

But, those instructions should be contained within the descriptions for the arguments.

**Comment by TerryGLee** [comment]
@MisinformedDNA @cpdohert Thanks to you both for your feedback. We do have a list of examples on this page: [Command-line parameter examples for Visual Studio 2017 installation](https://docs.microsoft.com/en-us/visualstudio/install/command-line-parameter-examples). Meanwhile, we'll see what we can do to make the description on this page clearer.

**Comment by TerryGLee** [comment]
@MisinformedDNA @cpdohert Thanks for your feedback and apologies for the delay in closing out this issue that's been in our backlog for a while. FYI: We've since added a sentence to the "`--add <one or more workload or component IDs>`" entry that says "To include multiple workloads or components, repeat the `--add` command (for example, `--add Workload1` `--add Workload2`)." I've also reformatted the Note in the middle of the page so that it is easier to see. The new Note formatting should be published by noon on Monday.
